!SESSION 2017-12-04 19:44:00.317 -----------------------------------------------
eclipse.buildId=2014.2
java.version=1.7.0_40
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -data C:/Users/sweuser/MyMultwithTrojan/MyMultwithTrojan.sdk

!ENTRY org.eclipse.osgi 2 0 2017-12-04 19:45:41.808
!MESSAGE One or more bundles are not resolved because the following root constraints are not resolved:
!SUBENTRY 1 org.eclipse.osgi 2 0 2017-12-04 19:45:41.808
!MESSAGE Bundle update@plugins/com.xilinx.sdk.managedbuilder_1.0.0.jar was not resolved.
!SUBENTRY 2 com.xilinx.sdk.managedbuilder 2 0 2017-12-04 19:45:41.808
!MESSAGE Missing required bundle org.slf4j.api.logback_1.5.8.
!SUBENTRY 1 org.eclipse.osgi 2 0 2017-12-04 19:45:41.808
!MESSAGE Bundle update@plugins/com.xilinx.sdk.debug.core_1.0.0.jar was not resolved.
!SUBENTRY 2 com.xilinx.sdk.debug.core 2 0 2017-12-04 19:45:41.808
!MESSAGE Missing required bundle org.slf4j.api.logback_1.5.8.
!SUBENTRY 1 org.eclipse.osgi 2 0 2017-12-04 19:45:41.808
!MESSAGE Bundle update@plugins/com.xilinx.sdk.debug.ui_1.0.0.jar was not resolved.
!SUBENTRY 2 com.xilinx.sdk.debug.ui 2 0 2017-12-04 19:45:41.808
!MESSAGE Missing required bundle org.slf4j.api.logback_1.5.8.

!ENTRY org.eclipse.osgi 2 0 2017-12-04 19:45:41.826
!MESSAGE The following is a complete list of bundles which are not resolved, see the prior log entry for the root cause if it exists:
!SUBENTRY 1 org.eclipse.osgi 2 0 2017-12-04 19:45:41.826
!MESSAGE Bundle com.xilinx.sdk.debug.core_1.0.0 [620] was not resolved.
!SUBENTRY 2 com.xilinx.sdk.debug.core 2 0 2017-12-04 19:45:41.826
!MESSAGE Missing required bundle org.slf4j.api.logback_1.5.8.
!SUBENTRY 2 com.xilinx.sdk.debug.core 2 0 2017-12-04 19:45:41.826
!MESSAGE Missing optionally required bundle net.sourceforge.cobertura_1.9.2.
!SUBENTRY 1 org.eclipse.osgi 2 0 2017-12-04 19:45:41.826
!MESSAGE Bundle com.xilinx.sdk.debug.ui_1.0.0 [623] was not resolved.
!SUBENTRY 2 com.xilinx.sdk.debug.ui 2 0 2017-12-04 19:45:41.826
!MESSAGE Missing required bundle org.slf4j.api.logback_1.5.8.
!SUBENTRY 2 com.xilinx.sdk.debug.ui 2 0 2017-12-04 19:45:41.826
!MESSAGE Missing optionally required bundle net.sourceforge.cobertura_1.9.2.
!SUBENTRY 1 org.eclipse.osgi 2 0 2017-12-04 19:45:41.826
!MESSAGE Bundle com.xilinx.sdk.managedbuilder_1.0.0 [634] was not resolved.
!SUBENTRY 2 com.xilinx.sdk.managedbuilder 2 0 2017-12-04 19:45:41.826
!MESSAGE Missing optionally required bundle net.sourceforge.cobertura_1.9.2.
!SUBENTRY 2 com.xilinx.sdk.managedbuilder 2 0 2017-12-04 19:45:41.826
!MESSAGE Missing required bundle org.slf4j.api.logback_1.5.8.

!ENTRY org.eclipse.osgi 4 0 2017-12-04 19:45:41.827
!MESSAGE Application error
!STACK 1
org.eclipse.swt.SWTException: Failed to execute runnable (java.lang.NullPointerException)
	at org.eclipse.swt.SWT.error(SWT.java:4361)
	at org.eclipse.swt.SWT.error(SWT.java:4276)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:138)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:4144)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3761)
	at org.eclipse.swt.widgets.Display.release(Display.java:3814)
	at org.eclipse.swt.graphics.Device.dispose(Device.java:295)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:140)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:110)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:79)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:353)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:180)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:629)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:584)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1438)
Caused by: java.lang.NullPointerException
	at com.xilinx.sdk.hw.HwProject.selectAndOpenHwSpec(HwProject.java:394)
	at com.xilinx.sdk.startup.InitSDK$12.run(InitSDK.java:450)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:35)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:135)
	... 17 more

!ENTRY org.eclipse.core.jobs 2 2 2017-12-04 19:45:44.401
!MESSAGE Job found still running after platform shutdown.  Jobs should be canceled by the plugin that scheduled them during shutdown: org.eclipse.ui.internal.Workbench$63

!ENTRY org.eclipse.core.jobs 2 2 2017-12-04 19:45:44.403
!MESSAGE Job found still running after platform shutdown.  Jobs should be canceled by the plugin that scheduled them during shutdown: com.xilinx.sdk.startup.InitSDK$6

!ENTRY org.eclipse.core.jobs 2 2 2017-12-04 19:45:44.404
!MESSAGE Job found still running after platform shutdown.  Jobs should be canceled by the plugin that scheduled them during shutdown: org.eclipse.ui.internal.progress.ProgressMonitorFocusJobDialog$3
!SESSION 2017-12-04 19:46:22.202 -----------------------------------------------
eclipse.buildId=2014.2
java.version=1.7.0_40
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -data C:/Users/sweuser/MyMultwithTrojan/MyMultwithTrojan.sdk

!ENTRY org.eclipse.core.resources 2 10035 2017-12-04 19:46:23.935
!MESSAGE The workspace exited with unsaved changes in the previous session; refreshing workspace to recover changes.

!ENTRY org.eclipse.cdt.core 1 0 2017-12-04 19:49:52.453
!MESSAGE Indexed 'TestTrojan' (2 sources, 25 headers) in 1.55 sec: 961 declarations; 1,291 references; 0 unresolved inclusions; 0 syntax errors; 0 unresolved names (0.00%)

!ENTRY org.eclipse.cdt.core 1 0 2017-12-04 19:49:57.596
!MESSAGE Indexed 'TestTrojan_bsp' (89 sources, 110 headers) in 3.46 sec: 3,609 declarations; 14,568 references; 59 unresolved inclusions; 9 syntax errors; 38 unresolved names (0.21%)

!ENTRY org.eclipse.core.jobs 4 2 2017-12-04 20:57:30.426
!MESSAGE An internal error occurred during: "Launching New_configuration".
!STACK 0
java.lang.RuntimeException: No Target with ID 64 in the System


	at com.xilinx.sdk.targetmanager.internal.TM.stop(TM.java:830)
	at com.xilinx.sdk.debug.core.internal.AppRunner.stopRun(AppRunner.java:192)
	at com.xilinx.sdk.debug.core.internal.RemoteProcess.terminate(RemoteProcess.java:132)
	at org.eclipse.debug.core.Launch.terminate(Launch.java:252)
	at com.xilinx.sdk.debug.core.XilinxAppLaunchConfigurationDelegate.launch(XilinxAppLaunchConfigurationDelegate.java:266)
	at com.xilinx.sdk.debug.ui.XilinxAppLaunchDelegateWrapper.launch(XilinxAppLaunchDelegateWrapper.java:31)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:855)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:704)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1047)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1251)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)

!ENTRY org.eclipse.core.jobs 4 2 2017-12-04 20:59:24.928
!MESSAGE An internal error occurred during: "Launching New_configuration".
!STACK 0
java.lang.RuntimeException: No Target with ID 64 in the System


	at com.xilinx.sdk.targetmanager.internal.TM.stop(TM.java:830)
	at com.xilinx.sdk.debug.core.internal.AppRunner.stopRun(AppRunner.java:192)
	at com.xilinx.sdk.debug.core.internal.RemoteProcess.terminate(RemoteProcess.java:132)
	at org.eclipse.debug.core.Launch.terminate(Launch.java:252)
	at com.xilinx.sdk.debug.core.XilinxAppLaunchConfigurationDelegate.launch(XilinxAppLaunchConfigurationDelegate.java:266)
	at com.xilinx.sdk.debug.ui.XilinxAppLaunchDelegateWrapper.launch(XilinxAppLaunchDelegateWrapper.java:31)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:855)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:704)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1047)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1251)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)

!ENTRY org.eclipse.core.jobs 4 2 2017-12-04 21:08:20.140
!MESSAGE An internal error occurred during: "Launching New_configuration".
!STACK 0
java.lang.RuntimeException: No Target with ID 64 in the System


	at com.xilinx.sdk.targetmanager.internal.TM.stop(TM.java:830)
	at com.xilinx.sdk.debug.core.internal.AppRunner.stopRun(AppRunner.java:192)
	at com.xilinx.sdk.debug.core.internal.RemoteProcess.terminate(RemoteProcess.java:132)
	at org.eclipse.debug.core.Launch.terminate(Launch.java:252)
	at com.xilinx.sdk.debug.core.XilinxAppLaunchConfigurationDelegate.launch(XilinxAppLaunchConfigurationDelegate.java:266)
	at com.xilinx.sdk.debug.ui.XilinxAppLaunchDelegateWrapper.launch(XilinxAppLaunchDelegateWrapper.java:31)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:855)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:704)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1047)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1251)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)

!ENTRY org.eclipse.core.jobs 4 2 2017-12-04 21:13:59.388
!MESSAGE An internal error occurred during: "Launching New_configuration".
!STACK 0
java.lang.RuntimeException: No Active System

	at com.xilinx.sdk.targetmanager.internal.TM.stop(TM.java:830)
	at com.xilinx.sdk.debug.core.internal.AppRunner.stopRun(AppRunner.java:192)
	at com.xilinx.sdk.debug.core.internal.RemoteProcess.terminate(RemoteProcess.java:132)
	at org.eclipse.debug.core.Launch.terminate(Launch.java:252)
	at com.xilinx.sdk.debug.core.XilinxAppLaunchConfigurationDelegate.launch(XilinxAppLaunchConfigurationDelegate.java:266)
	at com.xilinx.sdk.debug.ui.XilinxAppLaunchDelegateWrapper.launch(XilinxAppLaunchDelegateWrapper.java:31)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:855)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:704)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1047)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1251)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)

!ENTRY org.eclipse.core.jobs 4 2 2017-12-04 21:15:45.594
!MESSAGE An internal error occurred during: "Launching New_configuration".
!STACK 0
java.lang.RuntimeException: No Target with ID 64 in the System


	at com.xilinx.sdk.targetmanager.internal.TM.stop(TM.java:830)
	at com.xilinx.sdk.debug.core.internal.AppRunner.stopRun(AppRunner.java:192)
	at com.xilinx.sdk.debug.core.internal.RemoteProcess.terminate(RemoteProcess.java:132)
	at org.eclipse.debug.core.Launch.terminate(Launch.java:252)
	at com.xilinx.sdk.debug.core.XilinxAppLaunchConfigurationDelegate.launch(XilinxAppLaunchConfigurationDelegate.java:266)
	at com.xilinx.sdk.debug.ui.XilinxAppLaunchDelegateWrapper.launch(XilinxAppLaunchDelegateWrapper.java:31)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:855)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:704)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1047)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1251)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)

!ENTRY org.eclipse.core.jobs 4 2 2017-12-04 21:17:46.008
!MESSAGE An internal error occurred during: "Launching New_configuration".
!STACK 0
java.lang.RuntimeException: No Target with ID 64 in the System


	at com.xilinx.sdk.targetmanager.internal.TM.stop(TM.java:830)
	at com.xilinx.sdk.debug.core.internal.AppRunner.stopRun(AppRunner.java:192)
	at com.xilinx.sdk.debug.core.internal.RemoteProcess.terminate(RemoteProcess.java:132)
	at org.eclipse.debug.core.Launch.terminate(Launch.java:252)
	at com.xilinx.sdk.debug.core.XilinxAppLaunchConfigurationDelegate.launch(XilinxAppLaunchConfigurationDelegate.java:266)
	at com.xilinx.sdk.debug.ui.XilinxAppLaunchDelegateWrapper.launch(XilinxAppLaunchDelegateWrapper.java:31)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:855)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:704)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1047)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1251)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)

!ENTRY org.eclipse.core.jobs 4 2 2017-12-04 21:24:10.343
!MESSAGE An internal error occurred during: "Launching New_configuration".
!STACK 0
java.lang.RuntimeException: No Target with ID 64 in the System


	at com.xilinx.sdk.targetmanager.internal.TM.stop(TM.java:830)
	at com.xilinx.sdk.debug.core.internal.AppRunner.stopRun(AppRunner.java:192)
	at com.xilinx.sdk.debug.core.internal.RemoteProcess.terminate(RemoteProcess.java:132)
	at org.eclipse.debug.core.Launch.terminate(Launch.java:252)
	at com.xilinx.sdk.debug.core.XilinxAppLaunchConfigurationDelegate.launch(XilinxAppLaunchConfigurationDelegate.java:266)
	at com.xilinx.sdk.debug.ui.XilinxAppLaunchDelegateWrapper.launch(XilinxAppLaunchDelegateWrapper.java:31)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:855)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:704)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1047)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1251)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)

!ENTRY org.eclipse.debug.core 4 0 2017-12-04 21:33:41.016
!MESSAGE FrameworkEvent ERROR
!STACK 0
org.osgi.framework.BundleException: Exception in org.eclipse.debug.core.DebugPlugin.stop() of bundle org.eclipse.debug.core.
	at org.eclipse.osgi.framework.internal.core.BundleContextImpl.stop(BundleContextImpl.java:791)
	at org.eclipse.osgi.framework.internal.core.BundleHost.stopWorker(BundleHost.java:510)
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.suspend(AbstractBundle.java:566)
	at org.eclipse.osgi.framework.internal.core.Framework.suspendBundle(Framework.java:1206)
	at org.eclipse.osgi.framework.internal.core.StartLevelManager.decFWSL(StartLevelManager.java:592)
	at org.eclipse.osgi.framework.internal.core.StartLevelManager.doSetStartLevel(StartLevelManager.java:257)
	at org.eclipse.osgi.framework.internal.core.StartLevelManager.shutdown(StartLevelManager.java:215)
	at org.eclipse.osgi.framework.internal.core.InternalSystemBundle.suspend(InternalSystemBundle.java:284)
	at org.eclipse.osgi.framework.internal.core.Framework.shutdown(Framework.java:692)
	at org.eclipse.osgi.framework.internal.core.Framework.close(Framework.java:600)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.shutdown(EclipseStarter.java:399)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:199)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:629)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:584)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1438)
Caused by: java.lang.NullPointerException
Root exception:
java.lang.NullPointerException
!SESSION 2017-12-04 22:12:10.532 -----------------------------------------------
eclipse.buildId=2014.2
java.version=1.7.0_40
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -data C:/Users/sweuser/MyMultwithTrojan/MyMultwithTrojan.sdk

!ENTRY org.eclipse.ui 4 4 2017-12-05 15:06:11.445
!MESSAGE An internal error has occurred.
!STACK 0
java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
	at java.util.ArrayList.rangeCheck(ArrayList.java:635)
	at java.util.ArrayList.get(ArrayList.java:411)
	at java.util.Collections$SynchronizedList.get(Collections.java:1879)
	at org.eclipse.ui.internal.progress.TaskBarProgressManager$2.runInUIThread(TaskBarProgressManager.java:100)
	at org.eclipse.ui.progress.UIJob$1.run(UIJob.java:95)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:35)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:135)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:4144)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3761)
	at org.eclipse.ui.internal.Workbench.runEventLoop(Workbench.java:2701)
	at org.eclipse.ui.internal.Workbench.runUI(Workbench.java:2665)
	at org.eclipse.ui.internal.Workbench.access$4(Workbench.java:2499)
	at org.eclipse.ui.internal.Workbench$7.run(Workbench.java:679)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:332)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:668)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:149)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:124)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:110)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:79)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:353)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:180)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:629)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:584)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1438)
!SESSION 2017-12-06 17:50:53.345 -----------------------------------------------
eclipse.buildId=2014.2
java.version=1.7.0_40
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -data C:/Users/sweuser/MyMultwithTrojan/MyMultwithTrojan.sdk

!ENTRY org.eclipse.core.jobs 2 2 2017-12-08 11:45:05.809
!MESSAGE Job found still running after platform shutdown.  Jobs should be canceled by the plugin that scheduled them during shutdown: com.xilinx.sdk.hw.sync.HwPlatformMonitorUtils$1
